region/brinstar/red/Caterpillar Room.json

```
@@ -2214,6 +2323,27 @@
         "Morph"
       ],
       "flashSuitChecked": true
+    },
+    {
+      "link": [6, 5],
+      "name": "G-Mode Despawn the Gate, Open Gate",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "indirect",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "Morph",
+        "canOffScreenMovement"
+      ],
+      "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
+      "note": [
+        "Exit G-Mode while under the gate to open it without a Super.",
+        "The camera will not scroll through the morph tunnel, but Samus will still be visible;",
+        "position where Samus appears to be standing on air one tile to the right of the ledge."
```

Something like this could be added to the others as well, though Samus will only sometimes have morph with them.
```suggestion
        "position where Samus appears to be standing on air one tile to the right of the ledge.",
        "Samus can then use the morph tunnel to fix the camera."
```